src/components/Progress.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import './scss/Progress.scss';
 
 const Progress = () => (
-  <div>Progress</div>
+  <div className="progress-container d_flex_r g10">
+    <div className="circle-wrap">
+      <div className="circle" />
+    </div>
+    <div className="percentage_container">
+      <h3 className="percent-progress">100%</h3>
+      <p className="complete">Completed</p>
+    </div>
+  </div>
```

Comment:
- [ ] OPTIONAL: I recommend using CircularProgressbar library where you can easily setup the progress bar and set it to any percentage you want, here is an example: 

```
 <CircularProgressbar
                value={64}
                strokeWidth={8}
                styles={buildStyles({
                  strokeLinecap: 'butt',
                  pathTransitionDuration: 0.5,
                  pathColor: '#0290ff',
                  trailColor: '#DFE3E8',
                  backgroundColor: '#3e98c7',
                })}
              />
```

 value={64} represents 64% 😉